app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js

```diff
@@ -305,18 +298,9 @@ async function addEthereumChainHandler(
       properties: {
         chain_id: _chainId,
         network_name: _chainName,
-        // Including network to override the default network
-        // property included in all events. For RPC type networks
-        // the MetaMetrics controller uses the rpcUrl for the network
-        // property.
-        network: rpcUrlOrigin,
         symbol: ticker,
-        block_explorer_url: firstValidBlockExplorerUrl,
```

Comment:
based on https://github.com/MetaMask/metamask-extension/pull/16710/files#r1034677654 did we want to send this value as a boolean value? maybe `has_block_explorer_url` or `block_explorer_url_exists`? (not exactly sure what our naming conventions are for booleans here)

Reply:
I don't think knowing this serves us and i'd rather not further complicate this PR which is aimed at removing the offending values. Lets have further conversation about alternative tracking outside this PR and do follow up to figure out what is best.